spec/models/bathroom_spec.rb

```diff
@@ -6,4 +6,13 @@
   it { should validate_presence_of(:street_address) }
   it { should validate_presence_of(:zip_code) }
   it { should validate_presence_of(:description) }
+
+  context 'when search method called on bathroom' do
+    describe '.search' do
+      it 'should search zip codes' do
+        expect(Bathroom.search("02116")).to eq
+        Bathroom.where("zip_code = ?", "02116")
+      end
+    end
+  end
```

Comment:
What happens here when there are no matching results?

I would format your test blocks like this:

```
describe  ".search" do
  context "there are results" do
    # test
  end

  context "there are no matching results" do
    #test
  end
end
```